css-showcases/src/transition-helper.js

```diff
@@ -33,23 +27,25 @@ export const getTransitionsHtml = (element, props, mode) => {
     );
   }, 300);
 
+  const getAnimationMode = () => (mode === 'animation' ? 'no-anim' : '');
+
+  const boxWidth = '12rem';
+
   return /*html*/ `
-<div class="${styles.wrapper}">
+<div id="transitionBoxesContainer" class="${styles.boxesContainer}">
```

Comment:
why `id` here ?

Reply:
I was experimenting, not actually needed, I'll remove